examples/step-79/doc/intro.dox

```diff
@@ -6,23 +6,23 @@ structure that is bearing some load. Ideally, we would like to minimize the
 maximum stress placed on a structure by selecting a region $E$ where material is
 placed. In other words,
 @f[
-  \text{Minimize}\norm{\boldsymbol{\sigma} (\textbf{u}) }_\infty
+  \text{Minimize}\norm{\boldsymbol{\sigma} (\mathbf{u}) }_\infty
```

Comment:
We typically use `\mathbf{}` in preference to `\textbf{}` in math mode, so I made this change everywhere.